Dockerfile

```diff
@@ -45,10 +45,20 @@ RUN make build
 
 # Run image
 FROM quay.io/konveyor/move2kube:${VERSION}
+WORKDIR /move2kube-api
```

Comment:
why not use `workspace` or `workspaces`. The image by itself is for move2kube use only anyway.

Reply:
done